tests/Node.py

```diff
@@ -966,6 +966,30 @@ def publishContract(self, account, contractDir, wasmFile, abiFile, waitForTransB
         Node.validateTransaction(trans)
         return self.waitForTransBlockIfNeeded(trans, waitForTransBlock, exitOnError=False)
 
+    def installContract(self, contract):
+        contractDir = "contracts/%s" % (contract)
+        wasmFile = "%s.wasm" % (contract)
+        abiFile = "%s.abi" % (contract)
+        Utils.Print("Publish %s contract" % (contract))
+        trans = self.publishContract(contract, contractDir, wasmFile, abiFile, waitForTransBlock=True)
+        if trans is None:
+            Utils.errorExit("ERROR: Failed to publish contract %s." % contract)
+
+    def installStake(self, signer, waitForTransBlock=False, exitOnError=False):
+        self.installContract("cyber.govern")
```

Comment:
why is cyber.govern also installed here?

Reply:
it's for simplicity. not perfect name, I see.
for tests staking is only needed for schedule. we need govern too.
if there some resource-only test, we can separate installation of staking and govern or add flags